ts/features/services/home/components/FeaturedServiceCard.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  Avatar,
+  Badge,
+  H4,
+  IOColors,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import Placeholder from "rn-placeholder";
+import I18n from "../../../../i18n";
+import { WithTestID } from "../../../../types/WithTestID";
+import { logoForService } from "../utils";
+import OrganizationNameLabel from "./OrganizationNameLabel";
+
+export type FeaturedServiceCardProps = WithTestID<{
+  id: string;
+  name: string;
+  organization_name?: string;
+  accessibilityLabel?: string;
+  isNew?: boolean;
+  onPress?: () => void;
+}>;
+
+const styles = StyleSheet.create({
+  cardBaseContainer: {
+    height: 210,
+    width: 210,
+    padding: 16,
+    borderRadius: 8,
+    borderWidth: 1,
+    borderColor: IOColors["grey-100"],
+    backgroundColor: IOColors["grey-50"]
+  },
+  cardContainer: {
+    flex: 1,
+    flexDirection: "column",
+    alignItems: "flex-start",
+    justifyContent: "space-between"
+  },
+  cardContainerNew: {
+    borderColor: IOColors["hanPurple-100"],
+    backgroundColor: IOColors["hanPurple-50"]
+  },
+  cardHeader: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    width: "100%"
+  },
+  cardHeaderRight: {
+    alignSelf: "flex-start"
+  },
+  cardTitle: {
+    display: "flex",
+    flex: 1,
+    flexDirection: "column",
+    alignItems: "flex-end",
+    justifyContent: "flex-end"
```

Comment:
We could reduce complexity without affecting the component rendering:
```suggestion
  cardTitle: {
    width: "100%",
    marginTop: cardTitleMargin
```

Reply:
fe840b861